docs/architecture-3.0.md

```diff
@@ -13,11 +13,17 @@ The following architecture document addresses
   principles which got lost over the last years due to feature creep
   * making IntelMQ 3.x interoperable with CERT.pl's n6 system
 
+## Implementation evaluation
+Each topic listed in this document must go through an evaulation before the final implementation and integration into IntelMQ:
+- Evaluation of use-cases and demand
+- Evaluation of technical feasibility and implementation possibilities
+- Proof-of-concept and beta version, proofing positive impact in CERT's near-production environment
```

Comment:
Ich think that is too complex for now. I'd rather send it to the lists.

Reply:
true, I removed it